kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewContent.vue

```diff
@@ -0,0 +1,268 @@
+
+  export default {
+    name: 'PreviewContent',
+    components: {
+      ContentArea,
+      SlotTruncator,
+      LearningActivityIcon,
+      ResourceSelectionBreadcrumbs,
+    },
+    mixins: [commonCoreStrings, commonCoach],
+    setup() {
+      const {
+        addText$,
+        copyrightHolderDataHeader$,
+        licenseDataHeader$,
+        addedIndicator$,
+        notAvailableLabel$,
+      } = searchAndFilterStrings;
+
+      return {
+        addText$,
+        licenseDataHeader$,
+        copyrightHolderDataHeader$,
+        addedIndicator$,
+        notAvailableLabel$,
+      };
+    },
+    props: {
+      currentContentNode: {
+        type: Object,
+        required: true,
+      },
+      ancestors: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+      isSelected: {
+        type: Boolean,
+        required: true,
+      },
+      questions: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+    },
+    data() {
+      return {
+        selectedQuestionIndex: 0,
+      };
+    },
+    computed: {
+      channelsLink() {
+        return {
+          name: PageNames.LESSON_SELECT_RESOURCES_INDEX,
+        };
+      },
+      isExercise() {
+        return this.content.kind === ContentNodeKinds.EXERCISE;
+      },
+      selectedQuestion() {
+        if (this.isExercise) {
+          return this.questions[this.selectedQuestionIndex];
+        }
+        return '';
+      },
+      licenseName() {
+        return licenseLongName(this.content.license_name);
+      },
+      content() {
+        return this.currentContentNode;
+      },
+      description() {
+        if (this.content && this.content.description) {
+          const md = new markdownIt('zero', { breaks: true });
+          return md.render(this.content.description);
+        }
+
+        return undefined;
+      },
+      learningActivities() {
+        if (this.content.learning_activities) {
+          return this.content.learning_activities;
+        }
+        return [];
+      },
+    },
+    methods: {
+      topicsLink(topicId) {
+        const { params, query } = this.$route;
+
+        return {
+          name: PageNames.LESSON_SELECT_RESOURCES_TOPIC_TREE,
+          params: params,
+          query: {
+            ...query,
+            topicId,
+          },
+        };
+      },
+      questionLabel(questionIndex) {
+        if (!this.isExercise) {
+          return '';
+        }
+        const questionNumber = questionIndex + 1;
+        return this.coreString('questionNumberLabel', { questionNumber });
+      },
+      addResource() {
+        this.$emit('addResource', this.content);
+      },
+      removeResource() {
+        this.$emit('removeResource', this.content);
+      },
+      getTime(seconds) {
+        return this.$tr('minutes', { value: Math.floor(seconds / 60) });
+      },
+    },
+    $trs: {
+      minutes: {
```

Comment:
Should this string be in a strings file? (Non-blocking, just checking-in :sweat_smile:)

Reply:
Yes, let's put it there just for consistency